### DISCUSSION TITLE
Changes made on this repo since relase 1.0.6

### DISCUSSION THREAD

**Discussion by LaZsolt** [dbuezas/lgt8fx#191] [General]
Would you like, if I'll make a list of changes made on this repo since relase 1.0.6? It will look like this:

- Software serial timings revised [#34](https://github.com/dbuezas/lgt8fx/pull/34)
- Standard Arduino EEPROM API added [#64](https://github.com/dbuezas/lgt8fx/pull/64)
  Details: https://github.com/dbuezas/lgt8fx/wiki/EEPROM-of-the-LGT8F328p-:-the-important-things-you-need-to-know
- Added  LGT8F328P extended ISR vectors and definitions [#40](https://github.com/dbuezas/lgt8fx/pull/40)

**Comment by dbuezas** [comment]
Thanks @LaZsolt !
I'll also write down the exact deployment steps so the next one is not so dependant on my available time :)

**Comment by LaZsolt** [comment]
### Changes made on this repo since relase 1.0.6
  (This comment will be updated if needed.)

- This repo supports LGT8F328P and LGT8F328D only.
- Accelerated analogRead() by @jayzakk [#32](https://github.com/dbuezas/lgt8fx/pull/32), issue [#31](https://github.com/dbuezas/lgt8fx/issues/31)
- Software serial timings revised by @jg1uaa [#34](https://github.com/dbuezas/lgt8fx/pull/34)
- Corrected and added digital ports definitions (ports C6, E5, E6) by @jayzakk [#36](https://github.com/dbuezas/lgt8fx/pull/36), @LaZsolt [#185](https://github.com/dbuezas/lgt8fx/pull/185), issue [#174](https://github.com/dbuezas/lgt8fx/issues/174)
- Added LGT8F328's extended ISR vectors and definitions (eg. Timer3 ISR) by @jayzakk [#40](https://github.com/dbuezas/lgt8fx/pull/40)
- Definitions has been added for USART SPI by @jayzakk [#43](https://github.com/dbuezas/lgt8fx/pull/43)
- Faster SPI trasfer with 4 byte internal buffer by @jayzakk [#45](https://github.com/dbuezas/lgt8fx/pull/45)
- The global package name "LGT8fx Boards" in the packages .json file will change to "LGT8fx" _<b>(This issue is open.)</b>_ [#48](https://github.com/dbuezas/lgt8fx/issues/48)
- Improved accuracy in delayMicroseconds() by @LaZsolt [#51](https://github.com/dbuezas/lgt8fx/pull/51), [#74](https://github.com/dbuezas/lgt8fx/pull/74)
  Details: https://github.com/LaZsolt/delayMicroseconds/blob/master/for_LGT8F/readme.md
- Renew core and libraries (SPI,Wire) with ArduinoCore-1.8.3 by @jg1uaa [#63](https://github.com/dbuezas/lgt8fx/pull/63)
- Standard Arduino EEPROM API added by @SuperUserNameMan [#64](https://github.com/dbuezas/lgt8fx/pull/64), [#129](https://github.com/dbuezas/lgt8fx/pull/129)
  Details: https://github.com/dbuezas/lgt8fx/wiki/EEPROM-of-the-LGT8F328p-:-the-important-things-you-need-to-know
- Renew lgtx8x.h with Arduino style register definitions and completed with missing register definitions, like TCNCT3 for Timer3 by @LaZsolt [#79](https://github.com/dbuezas/lgt8fx/pull/79), issue [#57](https://github.com/dbuezas/lgt8fx/issues/57)
- Clock selection menu redesigned because of some boars have external XTAL by @jg1uaa _<b>(This PR is open.)</b>_ [#119](https://github.com/dbuezas/lgt8fx/pull/119)
  Details: https://github.com/dbuezas/lgt8fx/pull/119#issuecomment-1006800753
- A8, A9, A10, A11 (depending on chip pin count) and LGT8Fx specific internal analog inputs (VCCM, V5D1, V5D4, DACO) are accessible.
  by @LaZsolt _<b>(This PR is open.)</b>_ [#130](https://github.com/dbuezas/lgt8fx/pull/130) Details: https://github.com/dbuezas/lgt8fx/discussions/123#discussioncomment-636040
- Change analogReadResolution() defaults to 10 bits for backward compatibility with AVR based boards by @LaZsolt _<b>(This PR is open.)</b>_ [#130](https://github.com/dbuezas/lgt8fx/pull/130)
- Safe save LGT8F328D reference voltage calibration value <b>(PR will be later)</b> Issue: [#156](https://github.com/dbuezas/lgt8fx/issues/156)
